packages/mdc-dialog/README.md

```diff
@@ -0,0 +1,169 @@
+# MDC Dialog
+
+The MDC Dialog component is a spec-aligned dialog component adhering to the
+[Material Design dialog pattern](https://material.google.com/patterns/navigation-dialog.html).
+It implements a modal dialog window. You may notice that full screen components outlined in the dialog spec
+do not appear in MDC Dialog. This is because they have been deemed to be outside of the scope of what
+a dialog should be.
+
+## Installation
+
+```
+npm install --save @material/dialog
+```
+
+## Dialog usage
+
+Dialogs inform users about a specific task and may contain critical information or require decisions.
+
+
+```html
+<aside id="my-mdc-dialog"
+  class="mdc-dialog"
+  role="alertdialog"
+  aria-hidden="true"
+  aria-labelledby="mdc-dialog__label"
+  aria-describedby="mdc-dialog__description">
+  <div class="mdc-dialog__surface">
+    <header class="mdc-dialog__header">
+      <h2 id="mdc-dialog__label" class="mdc-dialog__header__title">
+        Use Google's location service?
+      </h2>
+    </header>
+    <section id="mdc-dialog__description" class="mdc-dialog__body">
+      Let Google help apps determine location. This means sending anonymous location data to Google, even when no apps are running.
+    </section>
+    <footer class="mdc-dialog__footer">
+      <button type="button" class="mdc-button mdc-dialog__footer__button mdc-dialog--cancel">DECLINE</button>
+      <button type="button" class="mdc-button mdc-dialog__footer__button mdc-dialog--accept">ACCEPT</button>
+    </footer>
+  </div>
+  <div class="mdc-dialog__backdrop"></div>
+</aside>
+```
+
+In the example above, we've created a dialog box in an `aside` element. Note that you can place content inside 
+the dialog. There are two types: dialog & dialogs with scrollable content. These are declared using CSS classes.
+
+CSS classes:
```

Comment:
While these tables are useful, I feel that they're a bit redundant given the DOM example above. Since every part of our DOM is mandatory, it doesn't need to be outlined. 

However, I do think it's useful to mention the use of `id`s for the label and description. That being said, we don't want users to think that they have to use _specific_ IDs for those sections. So we should indicate that they need to use _some_ `id` on those elements and reference them using the proper `aria-*` attrs in order for the dialog to be accessible.